src/__tests__/ReactHardwareComponent-test.js

```diff
@@ -4,6 +4,11 @@ import ReactHardwareComponent from '../ReactHardwareComponent';
 import ReactHardwareReconcileTransaction from '../ReactHardwareReconcileTransaction';
 
 describe('ReactHardwareComponent', () => {
+  it('should warn if it gets an unknown type', () => {
+    // TODO: check so it is the correct error
```

Comment:
based on the previous comment this should be a warning so you’ll spyOn and `console.warn`